content/identity.md

```diff
@@ -0,0 +1,58 @@
+## XII. Identity
+
+### Use a short-lived workload identity to connect to backing services
+
+#### 1. A twelve-factor app uses a connection-specific, short-lived workload identity to connect to backing services.
+
+A twelve-factor app is provided with a dynamic, ephemeral identity that is
+specific to each connected [backing service](./backing-services.md). This
+identity is delivered via a well-known location (e.g., a file or known network
+location) and is short-lived to minimize the risk of compromise.
+
+##### Examples
+
+- An app retrieves its connection-specific identity from a file like
+  `/var/run/identity/token` referenced by an environment variable (e.g.,
+  `DEFAULT_IDENTITY`).
+
+##### Guidance
+
```

Comment:
This focuses on credentials, should we also talk about other aspects of identity 1) identifier (i.e. instance id) 2) policies, what can it access as a result of it's role

Reply:
i wanted to focus this primarily on creds but i'm not opposed to adding information about that if you want to propose some additions